_assets/javascripts/lib/set-redirect-url.js

```diff
@@ -8,12 +8,23 @@ window.onload = function() {
       .split(";")
       .find(selectedCookie => selectedCookie.startsWith(" " + cookieName));
     var isCorrectName = cookieValue ? cookieValue.includes(domainValue) : false;
+    var subDomain = filterForCorrectSubdomain(CRDS.env.environment);
 
     if (isCorrectName == true || cookieValue == undefined) {
       document.cookie =
         cookieName +
         encodeURIComponent(window.location.href) +
-        ";domain=.crossroads.net;path=/";
+        ";domain=" +
+        subDomain +
+        ".crossroads.net;path=/";
+    }
+
+    function filterForCorrectSubdomain(subDomainPrefix) {
+      if (subDomainPrefix == "") {
+        return "www";
+      } else if (subDomainPrefix == "development") {
+        return "local";
+      }
```

Comment:
I'm a fan of this being in a function, but I'm also a fan of ternary operations.

`return subDomainPrefix ? subDomainPrefix : 'www'` should do the trick.

Also, we don't want to change development to local. I think we would just set `CRDS.env.environment` to `local` and it would just work. This should be configurable as is.

Reply:
Thanks Jason ^ I didn't want to mess with CRDS.env.environment that uses 'development' already since I don't the impact this would have across the application . . . that being said, if it is better to do that then I can refactor this for sure . . . getting Jimmy's opinion on this as well.